### DISCUSSION TITLE
【读书会】知识图谱

### DISCUSSION THREAD

**Discussion by jiangzhonglian** [apachecn/home#228] [Ideas]
# <center>【读书会】知识图谱<center>

* 【推荐】知识图谱
* 知识图谱文档: <http://pelhans.com/2018/03/15/xiaoxiangkg-note1>
* 知识图谱视频: <https://www.bilibili.com/video/av25435535>

> 参与

* QQ群: 713436582
* 时间周期: 2018-08-20～2018-09-02 （共 14 天）
* 学习方式: 自主留言进度，无监督模式（主要是先过一遍，有一个全面的认识，笔记有时间的可以做）

> 内容

* P1 01.第一课： 知识图谱概论
* P2 02.第二课：知识表示与知识建模
* P3 03.第三课：知识抽取与挖掘I
* P4 04.第四课：知识抽取与挖掘II
* P5 05.第五课：知识存储
* P6 06.第六课：知识融合
* P7 07.第七课：知识推理
* P8 08.第八课：语义搜索
* P9 09.第九课：知识问答I
* P10 10.第十课：IBM watson Lite
* P11 11.第十一课：行业知识图谱应用

**Comment by wnma3mz** [comment]
2018-08-17 第1课
2018-08-18 第2课
2018-08-19 第3课
2018-08-20 第4课
2018-08-21 第5课
2018-08-22 摸鱼。。。
2018-08-23 第6课
2018-08-24 第7课
2018-08-25 第8课
2018-08-26 第9课
2018-08-27 第10课
2018-08-28 摸鱼。。。
2018-08-29 第11课
finish
等这段时间忙完，看情况。。。整理一份大纲QAQ

**Comment by jiangzhonglian** [comment]
2018-08-18  第1课
2018-08-19  第2课
2018-08-20  第3课
2018-08-21  第4课
2018-08-22 休息
2018-08-23  第5课

**Comment by xiangking** [comment]
2018-08-20 第1课：知识有点多，囫囵吞枣了一遍

**Comment by kunty9103** [comment]
8.20，第一课
8.21，第二课

**Comment by xiangking** [comment]
第二天打卡第二课，这个视频的信息量真可怕，不过第二章看到代码莫名有点开心

**Comment by xiangking** [comment]
第三天打卡第三课，大致上来说知识图谱还是一个庞大的工程，前端的数据获取、清理和处理，然后再用各种方法得到关系、事件等知识，并得到联系，最后放入知识库中。这一期看完，总算知道大致流程，感觉其实本质上和平常的工程差不多，只是由于需要获得知识，所以很多平常的工具都做了新包装，使得其可以处理知识。

**Comment by kunty9103** [comment]
8.22，第三课

**Comment by xiangking** [comment]
第四天打卡第四课，没时间实践下代码，只好过一下视频

**Comment by jiangzhonglian** [comment]
> @xiangking
> 第四天打卡第四课，没时间实践下代码，只好过一下视频

后面都是惊喜，哈哈  别慌！！

**Comment by kunty9103** [comment]
第四课，不容易

**Comment by XU-ZHOU** [comment]
抱大腿

**Comment by xiangking** [comment]
打卡第五课第六课，这么多章下来整理好逻辑后感觉越看越清晰，不过这个老师确实给的框架选择太多了，短时间实践不来
